Dockerfile

```diff
@@ -94,6 +94,16 @@ rm -rf mediawiki/.git
 RUN git clone --depth=1 https://github.com/wikimedia/mediawiki-skins-Vector -b ${WMF_BRANCH} Vector &&\
 rm -rf Vector/.git
 
+# other skins
+RUN git clone --depth=1 https://github.com/ProfessionalWiki/chameleon chameleon &&\
```

Comment:
Chameleon installation is usually done via Composer, since it depends on the [Bootstrap](https://github.com/ProfessionalWiki/Bootstrap) extension, which in turns depends on the [SCSS library](https://github.com/ProfessionalWiki/SCSS/). If you need to install Chameleon via git then you'll need to manually fetch the rest, or at least include Chameleon's composer.json somewhere like in composer.local.json.